Documentation/Signoffs/User-Interface-npatel45.md

```diff
@@ -0,0 +1,118 @@
+# Vending Machine Subsystem - User-Interface
+### Function of the Subsystem:
+---
+When designing vending machine the main component of the machine is to get input from the user and process that information and send that information to the rest of the parts corresponding to it. So, in the same way for ECE board vending machine the student must be able to input their information, and select products from given set of  choices so to target this requirement, UI (User Interface) subsystem is required.
+  
+•	Receive information from student using medium of a touch screen LCD (Liquid Crystal Display)
+ 
+### Specifications & Constraints:
```

Comment:
These constraints seem to not cover things that are important and instead say things that shouldn't need to be said (recognize depression of all keys?)

Reply:
Without a clear understanding of the system's constraints how can it be designed and analyzed?